ddtrace/settings/config.py

```diff
@@ -273,6 +274,10 @@ def __init__(self):
         )
         self._data_streams_enabled = asbool(os.getenv("DD_DATA_STREAMS_ENABLED", False))
         self._appsec_enabled = asbool(os.getenv(APPSEC_ENV, False))
+        self._automatic_login_events_mode = os.getenv(APPSEC.AUTOMATIC_USER_EVENTS_TRACKING, "extended")
```

Comment:
why is this defaulting to extended when we say the default is safe? Am I getting something mixed up? I couldn't find a test demonstrating what we wanted.

Reply:
Bug. I must have messed on some rebase :( There are some tests, but they test that safe it's the default when there is not value (which won't happen here) or has any other value, so this one escaped. Fixed on next commit.